templates/easyappointments/0/rancher-compose.yml

```diff
@@ -0,0 +1,75 @@
+version: '2'
+
+catalog:
+  name: 'Easy!Appointments'
+  version: 0.1.0
+  description: 'Open source appointment scheduler'
+  questions:
+    - variable: base_url
+      label: 'Base URL'
+      type: string
+      required: true
+    - variable: timezone
+      label: Timezone
+      type: string
+      default: UTC
+      required: true
+    - variable: version
+      label: Version
+      type: enum
+      default: 1.3.0
+      options:
+        - 1.3.0
+        - 1.3
+        - 1
+        - 1.2.0
+        - 1.2
+        - latest
```

Comment:
Use generic docker image tags, implicit or explicit, is a bad practice due to is not reproducible. Otherwise, catalog packages manage versions, update,... 

Could you please add one catalog package version for each docker image tag, one for `1.2.0` other for `1.3.0` and remove `latest`??

Reply:
Ok, sure